packages/preset-react-app/docs/getting-started/1.md

```diff
@@ -11,9 +11,11 @@ These lines will install Phenomic and its [react app preset](https://github.com/
 
 ⚠️ _If you don't have a ``package.json`` you can do this simple command to get
  a valid one:_
- 
+
 ```console
 echo '{"private":true}' > package.json
+# or you can also do
+yarn init
```

Comment:
I think instead of  `echo '{"private":true}' > package.json` we could just have `npm init` cause if the user installed `Phenomic` he is going to have npm anyways. We could remove the `yarn init` in that case.